integrations/python/pymongo/run-for-all.sh

```diff
@@ -0,0 +1,9 @@
+for file in ../../../tests/*
```

Comment:
Make this the first line:
```
#!/bin/bash -ex
```

Also please add a comment that explains what this file is supposed to do and why it's useful.

Reply:
Done.